go/libkb/util_windows.go

```diff
@@ -113,9 +114,63 @@ func SafeWriteToFile(g SafeWriteLogger, t SafeWriter, mode os.FileMode) error {
 	return err
 }
 
+func copyFile(src string, dest string) error {
+	srcFile, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer srcFile.Close()
+
+	destFile, err := os.Create(dest) // creates if file doesn't exist
```

Comment:
hmm maybe we shouldn't do that?
OTOH, it will already never do the repairman if config.json is already in the destination, and with this change, config.json should never be in the destination if the move fails, so I think this is OK.